src/ai/backend/storage/api/manager.py

```diff
@@ -505,6 +521,7 @@ async def init_manager_app(ctx: Context) -> web.Application:
     app.router.add_route("POST", "/folder/clone", clone_vfolder)
     app.router.add_route("GET", "/folder/mount", get_vfolder_mount)
     app.router.add_route("GET", "/volume/performance-metric", get_performance_metric)
+    app.router.add_route("GET", "/volume/statistics", get_statistics)
```

Comment:
I think the naming "statistics" is too broad, so that's why I've added `get_fs_usage()` abstract method. (But missed to add the URL handler)